src/Connection.php

```diff
@@ -78,28 +80,32 @@ public function __construct(array $config)
     /**
      * Begin a fluent query against a database collection.
      *
+     * @deprecated since mongodb/laravel-mongodb 4.8, use the function table() instead
+     *
      * @param  string $collection
      *
      * @return Query\Builder
      */
     public function collection($collection)
     {
-        $query = new Query\Builder($this, $this->getQueryGrammar(), $this->getPostProcessor());
+        @trigger_error('Since mongodb/laravel-mongodb 4.8, the method Connection::collection() is deprecated and will be removed in version 5.0. Use the function table() instead.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('Since mongodb/laravel-mongodb 4.8, the method Connection::collection() is deprecated and will be removed in version 5.0. Use the table() method instead.', E_USER_DEPRECATED);
```